core/callitopservice.class.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @since 1.3.0
+ */
+class CallItopService
+{
+	public function CallItopViaHttp($sUri, $aAdditionalData, $iTimeOut = -1)
+	{
+		$iCurrentTimeOut = ($iTimeOut !== -1) ? $iTimeOut : (int)Utils::GetConfigurationValue('itop_synchro_timeout', 600); // timeout in seconds, for a synchro to run
```

Comment:
Would it make sense to reverse the arguments?
```suggestion
		$iCurrentTimeOut = ($iTimeOut === -1) ? (int)Utils::GetConfigurationValue('itop_synchro_timeout', 600) : $iTimeOut; // timeout in seconds, for a synchro to run
```

Reply:
+1